packages/expo-asset/src/__tests__/Asset-test.ts

```diff
@@ -10,18 +8,23 @@ jest.mock('expo-file-system', () => {
   };
 });
 
+const { Platform } = jest.requireActual('expo-modules-core');
+
 jest.mock('expo-modules-core', () => {
-  const ModulesCore = jest.requireActual('expo-modules-core');
+  const UnimodulesCore = jest.requireActual('expo-modules-core');
```

Comment:
AFAIK we want to move away from the "unimodules" name

```suggestion
  const ExpoModulesCore = jest.requireActual('expo-modules-core');
```